src/content/docs/infrastructure/manage-your-data/data-instrumentation/apm-data-infrastructure-monitoring.mdx

```diff
@@ -88,7 +122,7 @@ When your <InlinePopover type="apm" /> and infrastructure data is linked, you ca
 
 When your <InlinePopover type="apm" /> and infrastructure data is linked, you have more available host data in APM.
 
-The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page contains a table with data about your app's hosts and instances, including:
+The APM [Summary](/docs/apm/applications-menu/monitoring/apm-overview-page) page contains a **Infrastructure** section with data about your app's hosts and instances, including:
```

Comment:
i may be misundesrtanding, but this seems a bit redundant at this point: haven't we already discussed this? also, more generally, I think it's overdocumenting to simply include a list of what's in the UI: this is redundant (we could just say 'provides data about' and give a general description. Also creates maintenance issues (if the exist list of metrics changes). So we avoid unnecessary overdocumentation and futureproof the docs more when we're more vague and just give a general description versus exact description.

Reply:
agree!